packages/bitcore-lib/lib/transaction/transaction.js

```diff
@@ -632,48 +632,66 @@ Transaction.prototype.from = function(utxo, pubkeys, threshold, opts) {
   return this;
 };
 
-Transaction.prototype._fromNonP2SH = function(utxo) {
+Transaction.prototype.associateInputs = function(utxos, pubkeys, threshold, opts) {
```

Comment:
Could you add documentation to .associateInputs? All public bitcore-lib have jsdoc formatted docs

Reply:
Does this look okay?